tests/wlr_layer_shell_v1.cpp

```diff
@@ -234,6 +225,79 @@ class LayerSurfaceAnchorTest:
 {
 };
 
+struct LayerLayerParams
+{
+    std::experimental::optional<zwlr_layer_shell_v1_layer> below;
+    std::experimental::optional<zwlr_layer_shell_v1_layer> above;
+};
+
+std::ostream& operator<<(std::ostream& os, const LayerLayerParams& layer)
+{
+    os << "layers:";
+    for (auto i : {layer.below, layer.above})
+    {
+        os << " ";
+        os << "Layer{";
+        if (i)
+        {
+            switch (i.value())
+            {
+            case ZWLR_LAYER_SHELL_V1_LAYER_BACKGROUND:
+                os << "background";
+                break;
+            case ZWLR_LAYER_SHELL_V1_LAYER_BOTTOM:
+                os << "background";
```

Comment:
It looks wrong to write the same text for all layers.

Reply:
Good catch